tests/cypress/integration/browser_langs_spec.js

```diff
@@ -0,0 +1,114 @@
+describe('Browser loads English', () => {
+  beforeEach(() => {
+    cy.clearCookie('okta_help_user_lang')
+  })
+
+  it('when no options provided (default)', () => {
+    cy.visit('okta_help.htm?id=csh-index')
+    cy.url().should('include', 'en-us/Content/index.htm')
+  })
+
+  it('when primary browser language is neither EN nor JA', () => {
```

Comment:
nit: "when browser language is invalid"

and then maybe we should add one more for "when browser language is unsupported"

Reply:
yes, that's clearer for intent, thanks! 
Added